### PR TITLE
Add workflow for automatic testing on pull requests

### PR DESCRIPTION
**Pull request by PackmanDude**

### CODE REVIEW

**Comment by PackmanDude** [comment]
We need to merge this into the main repository, so we can test if it actually works (I don't have another account to make pull requests on my own test repository).

**Comment by PackmanDude** [comment]
Probably, after this change bugs would be much easier and faster to find.

**Comment by PackmanDude** [comment]
I think we could move script's contents into the workflow run tag directly, but I am not sure (didn't read the docs).

**Comment by nykonhrytsyshyn** [comment]
honestly, I dont see the point of it yet, maybe it will come in handy someday 👍🏿

File: .github/scripts/test.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+set -eux
+
+javac src/test/java/com/minersstudios/Main.java
+output="$(java -cp src/test/java com.minersstudios.Main)"
+return test "$output" = 'Hello, PackmanDude!'
```
Comment (PackmanDude) [inline .github/scripts/test.sh:5]: I love pushing untested code online.

зображення

Comment (PackmanDude) [inline .github/scripts/test.sh:5]: I love pushing untested code online.

**Pull request closed without merging**